numpy/tests/test_scripts.py

```diff
@@ -62,32 +62,35 @@ def run_command(cmd, check_code=True):
 @pytest.mark.skipif(is_inplace, reason="Cannot test f2py command inplace")
 def test_f2py():
     # test that we can run f2py script
+
+    def try_f2py_commands(cmds):
```

Comment:
Out of scope for this change, but - this would make more sense as a `pytest.parametrize` rather than silencing the exceptions and counting.

Reply:
Yep, I think the test and supporting function can probably use some cleanup at some point. I probably needs some work from someone running Windows to check if that part can also be improved.